Framework/API/test/FunctionFactoryTest.h

```diff
@@ -433,9 +433,9 @@ class FunctionFactoryTest : public CxxTest::TestSuite {
   void test_MultiDomainFunction_creation_moreComplex() {
     const std::string fnString =
         "composite=MultiDomainFunction,NumDeriv=true;(name=FunctionFactoryTest_"
-        "FunctA,a0=0,a1=0.5;name=FunctionFactoryTest_FunctB,b0=0.1,b1=0.2,ties="
+        "FunctA,a0=0,a1=0.5;name=FunctionFactoryTest_FunctB,b0=0.1,ties="
```

Comment:
This string is used in the check at line 447 and the output of `asString()` has changed so it outputs tied parameters only in the `ties=(...)` section.